client/state/reader-ui/actions.js

```diff
@@ -13,3 +17,21 @@ export const viewStream = ( streamKey, path ) => ( {
 	path,
 	streamKey,
 } );
+
```

Comment:
noting we can add simple tests for the actions, selectors, reducer.

Reply:
Ah ok, I'll take a look

Reply:
Added tests - but just for the new reducer, actions and selectors added in this PR